lib/ansible/modules/system/aix_lvg.py

```diff
@@ -0,0 +1,437 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Kairo Araujo <kairo@kairo.eti.br>
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author: Kairo Araujo (@kairoaraujo)
+module: aix_lvg
+short_description: Configure LVM volume groups for AIX.
+description:
+  - This module creates, removes or resize volume groups on AIX LVM.
+version_added: "2.5"
+options:
+  force:
+    description:
+      - Forces volume group creation.
+    choices: ["yes", "no"]
+    default: "no"
+  pp_size:
+    description:
+      - Size of the physical partition in megabytes.
+    required: false
+  pvs:
+    description:
+      - List of comma-separated devices to use as physical devices in this
+        volume group. Required when creating or extending (C(present) state)
+        the volume group. If not informed reducing (C(absent) state) the volume
+        group will be removed.
+    required: false
+  state:
+    description:
+      - Control if the volume group exists and volume group AIX state varyonvg
+        C(varyon) or varyoffvg C(varyoff).
+    choices: [present, absent, varyon, varyoff]
+    default: present
+    required: false
+  vg:
+    description:
+      - Volume group name.
+    required: true
+  vg_type:
+    description:
+      - Volume group type.
+    choices: [normal, big, scalable]
+    default: normal
+    required: false
+notes:
+  - AIX will permit remove VG only if all LV/Filesystems aren't busy.
+  - Module does not modify PP size for already present volume group.
+'''
+
+EXAMPLES = '''
+- name: Create a volume group datavg
+  aix_lvg:
+    vg: datavg
+    pp_size: 128
+    vg_type: scalable
+    state: present
+
+- name: Removing a volume group datavg
+  aix_lvg:
+    vg: datavg
+    state: absent
+
+- name: Extending rootvg
+  aix_lvg:
+    vg: rootvg
+    pvs: hdisk1
+    state: present
+
+- name: Reducing rootvg
+  aix_lvg:
+    vg: rootvg
+    pvs: hdisk1
+    state: absent
+'''
+
+RETURN = '''
+changed:
+  description: Return changed for aix_lvg actions as true or false.
+  returned: always
+  type: boolean
+  version_added: 2.5
+msg:
+  description: Return message regarding the action.
+  returned: always
+  type: string
+  version_added: 2.5
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+
+# Command option parameters.
+force_opt = {
```

Comment:
I would put these variables into the function which is using them. No reason to have them as global vars if you don't share them across multiple functions.